packages/ui-kit/src/components/ThemeProvider/ThemeProvider.types.ts

```diff
@@ -39,7 +39,7 @@ export interface ThemeProviderProps extends Pick<ThemeContextProps, 'globalChart
 
 export type ThemeStateProps = ThemeTokenProps | undefined
 
-export type ChartVariant = 'bar' | 'line'
+export type ChartVariant = 'bar' | 'line' | 'pie' | 'doughnut'
```

Comment:
Does it make sense to put this here? How is ChartVariant being used? @hasancolak @sashathor @felipecadavid

Reply:
It seems like we are only using this ChartVariant in the ThemeProvider and the chartUtils helper, then TimeSeries has its own `TimeSeriesChartVariant` and for leaderboard `LeaderboardChartVariant`.

Not sure if we should expose `ChartVariant` in that case, that could be confusing.

Reply:
Since themeProvider providers the chart configuration it must be developed here. 
I extend it with other variant.
I do not have deep info about the scope of themeProvider should support that config or not?
Until clarifying the details I am leaving this at it is.